config.go

```diff
@@ -545,7 +545,7 @@ func (b *Builder) Comment() string {
 // discarded when writing images using OCIv1 formats.
 func (b *Builder) SetComment(comment string) {
 	if comment != "" && b.Format != define.Dockerv2ImageManifest {
-		logrus.Warnf("COMMENT is not supported for OCI image format, comment %s will be ignored. Must use `docker` format", comment)
+		b.Logger.Warnf("COMMENT is not supported for OCI image format, comment %s will be ignored. Must use `docker` format", comment)
```

Comment:
COMMENT is not an instruction, so this message is misleading.

Reply:
What should the message be?  Or is this actually impossible?

Reply:
Ok since this is only called via buildah config, I will just remove the change.  It does not look like buildah build or podman-remote build would ever hit this code.